packages/elements/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@stoplight/elements",
-  "version": "8.5.0",
```

Comment:
You need to bump up the `elements-core` version in the dependencies here and in `elements-dev-portal` as well.